cni/client/client.go

```diff
@@ -14,21 +14,25 @@ import (
 )
 
 const (
-	azureVnetBinName      = "./azure-vnet"
-	azureVnetBinDirectory = "/opt/cni/bin"
+	azureVnetExecutable = "/opt/cni/bin/azure-vnet"
```

Comment:
Does this client gets packaged with CNI too?

Reply:
no, it's only imported by cns